### DISCUSSION TITLE
Wireguard - Unable to access hosts on remote network

### DISCUSSION THREAD

**Discussion by mazarian** [tteck/Proxmox#488] [Help Needed]
Hello- on a newly created Wireguard LXC, I'm able to access the internet via the remote network over Wireguard, but not able to access hosts on the remote network. For example, I'm not able to access 192.168.1.5 but I am able to access google.com. I have confirmed that my traffic is being routed through the remote network. Any idea what might be the issue?

**Comment by mattvanmater** [comment]
I have the same problem.

E.g. I used the v4 version of the install script, LXC created with all the default choices.  Home network is 192.168.1.0/24 (LCX itself is 192.168.1.200).  The wg0 interface on the LXC is set to 10.6.0.1/24, client AllowedIP is set to 10.6.0.2/32

When I use the wireguard client on Android I am able to access the internet (non-local hosts) through the VPN but I can't access anything on the 192.168.1.0/24 network.  I think the issue is either the LXC needs NAT enabled outbound and/or I need static routes everywhere on my network pointing 10.6.0.0/24 to the internal IP of the LXC (192.168.1.200).  As far as I can tell there aren't a lot of logs available on either platform so I don't have much to troubleshoot with.  

My next troubleshooting steps are to look at raw pcaps on the LXC to see it the traffic is exiting the LXC as the 10.6.0.1 IP, and if there are any responses.  Hopefully someone else has resolved this and can save me some time.
